From 4354468de0dafffabb0ef082ac07e210ebfff26d Mon Sep 17 00:00:00 2001
From: Steve Arnold <sarnold@vctlabs.com>
Date: Sun, 18 Feb 2024 16:49:22 -0800
Subject: [PATCH 4/4] fix: dev: remove crypto_policy rules from standard
 profile

* packages currently exist only for RHEL variants, the last 2 ubuntu LTS
  releases, and debian sid
* revisit this at some future time (not required for pcr device)

Signed-off-by: Steve Arnold <sarnold@vctlabs.com>
---
 products/openembedded/profiles/standard.profile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/products/openembedded/profiles/standard.profile b/products/openembedded/profiles/standard.profile
index 2d3a785647..a614a4b885 100644
--- a/products/openembedded/profiles/standard.profile
+++ b/products/openembedded/profiles/standard.profile
@@ -125,8 +125,6 @@ selections:
     - require_singleuser_auth
     - require_emergency_target_auth
     - disable_users_coredumps
-    - configure_crypto_policy
-    - var_system_crypto_policy=default_policy
     - dir_perms_world_writable_sticky_bits
     - file_permissions_etc_passwd
     - file_owner_etc_shadow
-- 
2.43.0

